urbanopt_des/modelica_results.py

```diff
@@ -164,6 +350,22 @@ def resample_and_convert_to_df(
             building_data[f"ETS Thermal Cooling Building {building_id}"] = ets_q_cooling
             building_data[f"ETS Thermal Heating Building {building_id}"] = ets_q_heating
 
+        # Add in chiller aggregations
+        agg_columns["Chillers Total"] = []
```

Comment:
There doesn't appear to be an conditional logic necessary before these are created. Should we move them up a few lines to where agg_columns is created, or is it nicer to just instantiate those empty lists and make these later?

Reply:
I just wanted them to be where we are looking for the components.